### PR TITLE
Update 2021-10-29-kube-secrets-210011.adoc

### PR DESCRIPTION
**Pull request by yeekangc**
Fixes for typos

### CODE REVIEW

**Comment by yeekangc** [comment]
I feel the instruction included for this section on user feature can be further improved.  Then, the corresponding docs should cover it better and lay out what users and developers of user features should consider in a logical flow.  So, suggesting typo fixes for now.

@jjiwooLim @bensonlatibm

Cc @GraceJansen @gkwan-ibm

File: posts/2021-10-29-kube-secrets-210011.adoc [hunk 1]
```diff
@@ -84,10 +84,10 @@ For more information visit link:https://kubernetes.io/docs/concepts/configuratio
 [#user_features]
 == Enable installation of user features on Open Liberty
 
-The FeatureUtility command-line tool, Liberty Maven and Gradle Plugin will now enable installation of user features onto Open Liberty runtime from Maven Central and on-premise Maven repository. Prior to this update, users had to use `InstallUtility` command-line tool from commercial liberty.
+The `featureUtility` command-line tool, Liberty Maven and Gradle plugins will now enable installation of user features onto Open Liberty runtime from Maven Central and on-premises Maven repository.
```
Comment (dmuelle) [inline posts/2021-10-29-kube-secrets-210011.adoc]: suggested edit:

You can now install user features onto the Open Liberty runtime from Maven Central or on-premises Maven repository by using the `featureUtility` command-line tool and the Liberty Maven and Gradle plug-ins.

File: posts/2021-10-29-kube-secrets-210011.adoc [hunk 2]
```diff
@@ -114,7 +114,7 @@ Create `features-bom` (bill of Materials) file for the user feature. The `featur
 </project>
 ----
 
-Generate the `features.json` file with Liberty Maven or Gradle plugin. The `features.json` file is the `JSON` file that contains the information found within a feature's `ESA` manifest file. The `JSON` files are a key requirement for installation of any liberty features(s) from a maven repository. Provide the maven coordinate of the custom made `features-bom` file.
+Generate the `features.json` file with Liberty Maven or Gradle plugin. The `features.json` file is the `JSON` file that contains the information found within a feature's `ESA` manifest file. The `JSON` files are a key requirement for installation of any Liberty features(s) from a Maven repository. Provide the Maven coordinate of the custom made `features-bom` file.
```
Comment (dmuelle) [inline posts/2021-10-29-kube-secrets-210011.adoc]: suggested edit:
Generate the `features.json` file with the Liberty Maven or Gradle plugin. The `features.json` file is the `JSON` file that contains the information from a feature's `ESA` manifest file. The `JSON` files are a key requirement for installation of any Liberty feature from a Maven repository. Provide the Maven coordinate of the custom made `features-bom` file.

**Review by dmuelle — COMMENTED** [review]
a couple suggestions

**Comment by dmuelle** [comment]
@yeekangc I agree w/r/t the docs- we need a logical story around not just installing user features but developing them:
- what a user feature is and why you might want to develop them
- how to develop them
- how to install and maintain them  

We have two issues on our backlog for developing user features, [based on info from WL IBM docs](https://github.com/OpenLiberty/docs/issues/3324) and [an old WAS DEV post about developing them](https://github.com/OpenLiberty/docs/issues/2950). The info in IBM docs is spread across multiple topics and is fairly complex-  I'm also not sure how current it is at this point. We haven't been able to prioritize these issues yet and they would require significant engagement for an SME. But since we are now documenting how to install user features on Open Liberty I think we should develop the rest of the story.

**Comment by yeekangc** [comment]
@ryan-storey, can you handle these fixes ASAP?  Cc @mbroz2

File: posts/2021-10-29-kube-secrets-210011.adoc [hunk 3]
```diff
@@ -114,7 +114,7 @@ Create `features-bom` (bill of Materials) file for the user feature. The `featur
 </project>
 ----
 
-Generate the `features.json` file with Liberty Maven or Gradle plugin. The `features.json` file is the `JSON` file that contains the information found within a feature's `ESA` manifest file. The `JSON` files are a key requirement for installation of any liberty features(s) from a maven repository. Provide the maven coordinate of the custom made `features-bom` file.
+Generate the `features.json` file with the Liberty Maven or Gradle plugin. The `features.json` file is the `JSON` file that contains the information from a feature's `ESA` manifest file. The `JSON` files are a key requirement for installation of any Liberty feature from a Maven repository. Provide the Maven coordinate of the custom made `features-bom` file.
```
Comment (dmuelle) [inline posts/2021-10-29-kube-secrets-210011.adoc]: just noticed this- suggested edit

Provide the Maven coordinate of the custom made `features-bom` file.
--->
Provide the Maven coordinates of the `features-bom` file that you created for the feature.

File: posts/2021-10-29-kube-secrets-210011.adoc [hunk 4]
```diff
@@ -161,7 +161,7 @@ dependencies {
 }
 ----
 
-Install the user feature using the link:https://openliberty.io/docs/latest/reference/command/featureUtility-installFeature.html[`featureUtility`] command-line tool, Liberty Maven or Gradle plugin. Provide the maven coordinate of the custom made `features-bom` file.
+Install the user feature using the link:https://openliberty.io/docs/latest/reference/command/featureUtility-installFeature.html[`featureUtility`] command-line tool, Liberty Maven or Gradle plugin. Provide the Maven coordinate of the custom made `features-bom` file.
```
Comment (dmuelle) [inline posts/2021-10-29-kube-secrets-210011.adoc]: custom-made

Provide the Maven coordinate of the custom made features-bom file.
--->
Provide the Maven coordinates of the features-bom file that you created for the feature.

**Review by dmuelle — APPROVED** [review]
LGTM

**Comment by yeekangc** [comment]
@dmuelle @ryan-storey, I've merged to draft.  Let me know if I need to push to staging or the team will handle.  Thanks.

**Comment by dmuelle** [comment]
@yeekangc I opened https://github.com/OpenLiberty/blogs/pull/1904 to send the change to staging

**Pull request merged**